packages/tupaia-web/src/features/Visuals/Matrix/Matrix.tsx

```diff
@@ -174,7 +184,7 @@ const MatrixVisual = () => {
   }
 
   return (
-    <>
+    <Wrapper>
```

Comment:
In this case I think we do actually need an extra `div` - to make the table scrollable, the parent needs to have `overflow:hidden` and `max-height` set. Because the visuals share a parent (in this case, the modal) and we want only the table to scroll so that the header is sticky (as opposed to the entire modal content scrolling) I decided to add a div to only the matrix, so that it wouldn't have any effect on other visual types

Reply:
Apologies—I think you’re right! This must be one of the reasons MUI has a [`TableContainer`](https://mui.com/material-ui/api/table-container/) component